### PR TITLE
Ts-fix-move-students

### PR DESCRIPTION
**Pull request by Niknikson**

### CODE REVIEW

**Review by HiroFive — APPROVED** [review]

File: frontend/src/reducers/students.js [hunk 1]
```diff
@@ -31,6 +31,20 @@ const reducer = (state = initialState, action) => {
             };
         }
 
+        case actionTypes.DELETE_ALL_STUDENTS: {
+            const students = state.students.filter((student) => {
+                let res = true;
+                action.students.forEach((el) => {
```
Comment (ViktoriiaKosovan) [inline frontend/src/reducers/students.js]: maybe use some

**Review by ViktoriiaKosovan — APPROVED** [review]

**Review by Yashchemskyi-Oleksandr — APPROVED** [review]

File: frontend/public/assets/i18n/translations/en/formElements.json [hunk 2]
```diff
@@ -125,5 +125,6 @@
     "forgot_password_label": "Forgot password?",
     "full_name": "Full name",
     "dont_have_an_account": "Don't have an account?",
-    "reset_password_helper_text": "Just enter your email address below and we'll send you a link to reset your password!"
+    "reset_password_helper_text": "Just enter your email address below and we'll send you a link to reset your password!",
+    "students_was_label": "Students"
```
Comment (Akr1s) [inline frontend/public/assets/i18n/translations/en/formElements.json]: maybe reconsider this name

File: frontend/public/assets/i18n/translations/uk/formElements.json [hunk 3]
```diff
@@ -132,5 +132,6 @@
     "create_account_label": "Створити аккаунт",
     "forgot_password_label": "Забули пароль?",
     "dont_have_an_account": "Не маєте облікового запису?",
-    "reset_password_helper_text": "Просто введіть свою електронну адресу нижче, і ми надішлемо вам посилання для скидання пароля!"
+    "reset_password_helper_text": "Просто введіть свою електронну адресу нижче, і ми надішлемо вам посилання для скидання пароля!",
+    "students_was_label": "Студентів"
```
Comment (Akr1s) [inline frontend/public/assets/i18n/translations/uk/formElements.json]: As above

File: frontend/src/actions/students.js [hunk 4]
```diff
@@ -14,6 +14,11 @@ export const deleteStudentSuccess = (id) => ({
     id,
 });
 
+export const deleteAllStudentSuccess = (students) => ({
+    type: actionTypes.DELETE_ALL_STUDENTS,
+    students,
+});
+
```
Comment (Akr1s) [inline frontend/src/actions/students.js]: do we actually need a payload here?

File: frontend/src/constants/translationLabels/formElements.js [hunk 5]
```diff
@@ -123,3 +123,4 @@ export const DONT_HAVE_ACCOUNT_LABEL = 'formElements:dont_have_an_account';
 export const LOGIN_VIA_GOOGLE_LABEL = 'formElements:login_via_google';
 export const RESET_PASSWORD_HELPER_TEXT = 'formElements:reset_password_helper_text';
 export const FORM_CHOOSE_GROUP_LABEL = 'choose_group_title';
+export const STUDENTS_WAS_LABEL = 'formElements:students_was_label';
```
Comment (Akr1s) [inline frontend/src/constants/translationLabels/formElements.js]: Name

File: frontend/src/sagas/student.js [hunk 6]
```diff
@@ -113,11 +125,23 @@ function* uploadStudentsToGroup({ file, id }) {
...
     try {
-        const students = yield select(getStudents);
-        const movedStudents = students.filter((item) => item.checked === true);
-        console.log(movedStudents, group, newGroup);
+        const allStudents = yield select(getStudents);
+        const movedStudents = allStudents.filter((item) => item.checked === true);
+        const students = movedStudents.map((student) => {
+            const item = student;
+            delete item.checked;
+            return item;
```
Comment (Akr1s) [inline frontend/src/sagas/student.js]: Maybe use `omit` from lodash instead?

File: frontend/src/actions/students.js (hunk 4 above)
Comment (Niknikson) [reply frontend/src/actions/students.js]: > 

fixed  DELETE_SELECTED_STUDENTS
yes

File: frontend/src/constants/translationLabels/formElements.js (hunk 5 above)
Comment (Niknikson) [reply frontend/src/constants/translationLabels/formElements.js]: > students_uppercase
done

File: frontend/public/assets/i18n/translations/uk/formElements.json (hunk 3 above)
Comment (Niknikson) [reply frontend/public/assets/i18n/translations/uk/formElements.json]: > students_uppercase
done

File: frontend/public/assets/i18n/translations/en/formElements.json (hunk 2 above)
Comment (Niknikson) [reply frontend/public/assets/i18n/translations/en/formElements.json]: > students_uppercase
done

File: frontend/src/reducers/students.js (hunk 1 above)
Comment (Niknikson) [reply frontend/src/reducers/students.js]: > 

done

File: frontend/src/sagas/student.js (hunk 6 above)
Comment (Niknikson) [reply frontend/src/sagas/student.js]: > 

done

**Review by andrii-kuzmichov — APPROVED** [review]

**Pull request merged**